libs/mocks/src/outbound_queue.rs

```diff
@@ -3,11 +3,12 @@ pub mod pallet {
 	use cfg_traits::liquidity_pools::OutboundQueue;
 	use frame_support::pallet_prelude::*;
 	use mock_builder::{execute_call, register_call};
+	use orml_traits::GetByKey;
 
 	#[pallet::config]
 	pub trait Config: frame_system::Config {
-		type Sender;
-		type Destination;
+		type Sender: Parameter;
+		type Destination: Parameter;
```

Comment:
Why are these bounds needed in the mock?

Reply:
The compiler required both types to be encodable and decodable. I started with single traits but then just went with `Parameter` to have peace.

Reply:
Mmm, it's weird that the mock mandates those 🤔